cmd/utils.go

```diff
@@ -138,13 +129,44 @@ var generateDocsCmd = &cobra.Command{
 	},
 }
 
+// completionCmd generates auto completion commands
+var completionCmd = &cobra.Command{
+	Use:       "completion <shell>",
+	Hidden:    true,
+	ValidArgs: []string{"zsh", "bash"},
+	Args:      cobra.MinimumNArgs(1),
+	Short:     "Generates auto completion scripts",
+	Long: `To load completion add the following to your ~/.bashrc or ~/.zshrc file
+
+. <(corectl completion zsh)
+
+or
+
+. <(corectl completion bash)
+
+If you are running on OSX you will also need to install bash-completion e.g through brew.
```

Comment:
Redundant information, I understand that we want this to be accessible but it's easy to miss one place if we need to update the info, perhaps link to a section in README?